imls-backend/test/test_authentication.py

```diff
@@ -2,74 +2,65 @@
 from unittest import TestCase
 
+
 def endpoint(ep_arr):
     test_url = f"{os.getenv('POETRY_SCHEME')}://{os.getenv('POETRY_HOSTNAME')}:{os.getenv('POETRY_PORT')}"
     return test_url + "/" + "/".join(ep_arr)
 
+
 class LoginTests(TestCase):
     def test_call_login_invalid(self):
         url = endpoint(["rpc", "login"])
-        body = {
-            "fscs_id": "ME0018-001",
-            "api_key": "notapassword"
-        }
+        body = {"fscs_id": "ME0018-001", "api_key": "notapassword"}
         # Need to post, not get, if you're passing params in the body.
         r = requests.post(url, json=body)
         print(r.json())
         self.assertEqual(r.status_code, 403)
         # Should be invalid user or password error
-        self.assertEqual(r.json()['code'], "28P01")
+        self.assertEqual(r.json()["code"], "28P01")
 
     def test_login_successfully(self):
         url = endpoint(["rpc", "login"])
-        body = {
-            "fscs_id": "KY0069-002",
-            "api_key": "hello-goodbye"
-        }
+        body = {"fscs_id": "KY0069-002", "api_key": "hello-goodbye"}
         # Need to post, not get, if you're passing params in the body.
         r = requests.post(url, json=body)
         print("URL ", url)
         print("RESPONSE ", r.json())
         self.assertEqual(r.status_code, 200)
-        t0 = r.json()['token'];
+        t0 = r.json()["token"]
         # This comes back as three parts separated by periods.
         # blasdlkfjasdflkjas.balskdjfaskjdfhaksjdhf.baskdjhfaskdjhfashkldj
         # The first part of the token will always be the same.
         t1 = t0.split(".")[0]
         t2 = t0.split(".")[1]
         self.assertEqual(t1, "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9")
 
-    # Call "beat_the_heart" to see if we can get through authentication.    
+    # Call "beat_the_heart" to see if we can get through authentication.
     # CREATE OR REPLACE FUNCTION api.beat_the_heart(
-	# _fscs character varying,
-	# _sensor integer,
-	# _hb timestamp with time zone,
-	# _serial character varying,
-	# _version character varying)
+    # _fscs character varying,
+    # _sensor integer,
+    # _hb timestamp with time zone,
+    # _serial character varying,
+    # _version character varying)
     # RETURNS character varying
     def test_use_token(self):
         token_url = endpoint(["rpc", "login"])
-        body = {
-            "fscs_id": "KY0069-002",
-            "api_key": "hello-goodbye"
-        }
+        body = {"fscs_id": "KY0069-002", "api_key": "hello-goodbye"}
         # Need to post, not get, if you're passing params in the body.
         tr = requests.post(token_url, json=body)
         print("URL ", token_url)
         print("RESPONSE ", tr.json())
         self.assertEqual(tr.status_code, 200)
-        t0 = tr.json()['token'];
+        t0 = tr.json()["token"]
 
         url = endpoint(["rpc", "beat_the_heart"])
         body = {
-            "_fscs_id": "AA0005-001",
+            "_fscs_id": "KY0069-002",
```

Comment:
This was the only real change I made; my linter (`black`) automatically did the rest of the whitespace changes.